llisite/urls.py

```diff
@@ -3,9 +3,12 @@
 from django.contrib.staticfiles.urls import static
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 from django.conf import settings
+from django.contrib.flatpages import views
+# from django_markdown import flatpages
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
+    url(r'^(?P<url>.*/)$', views.flatpage),
```

Comment:
having this high up on the `urlpatterns` prevents the actual urls from other apps from working